src/input_utils.py

```diff
@@ -56,14 +61,37 @@ class PretrainInputConfig(object):
   # Maximum number of masked patch tokens per batch.
   mpp_max_selections_per_batch = attr.ib(default=1600)
 
+  # Maximum output channel bits in masked patch prediction.
+  output_channel_bits = attr.ib(default=3)
+
```

Comment:
What do you mean by 'channel bits'?

Reply:
3-bit channel means we use 0-7 to represent the channel. A normally image has 8-bit channels. Instead of predicting 8-bit channels, we predict 3-bit channels for masked patch prediction. By doing so, we can make number of output classes smaller (2^3)^3=512 classes.  I follow the ViT paper and [PyTorch implementation](https://github.com/lucidrains/vit-pytorch/blob/22da26fa4b7d98c2a936b48c1b1dcf83c404dfb8/vit_pytorch/mpp.py#L33).

Reply:
I see. Interesting.